### PR TITLE
Ticket-7: vote on an article

### PR DESCRIPTION
**Pull request by omobayed**

### CODE REVIEW

File: src/components/ArticleVotes.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { updateArticleVotes } from "../api"
+
+const ArticleVotes = ({ article_id, votes_count }) => {
+    const [userVotes, setUserVotes] = useState(0);
+    const [isError, setIsError] = useState(false)
+
+    const handleLikeClick = (event) => {
```
Comment (RichardNeat) [inline src/components/ArticleVotes.jsx]: These two functions look great and seem to be handling the optimistic rendering really well. Could we find a way of refactoring to make it more DRY? The only thing that is different is whether we go up by 1 or down by 1?

File: src/components/ArticleVotes.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            .catch((error) => {
+                setIsError(true)
+                setUserVotes((currentUserVote) => {
+                    return currentUserVote + 1;
+                })
+            })
+    }
+
+    return (
+        <div>
```
Comment (RichardNeat) [inline src/components/ArticleVotes.jsx]: Could we use anything better than a div here?

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice work so far on this one! Just a couple of points to look at in the comments below please.
Also could you take a look at your colours on the site, A lot of the writing is very difficult to read. Once these points have been addressed you can merge :)

File: src/components/ArticleVotes.jsx (hunk 2 above)
Comment (omobayed) [reply src/components/ArticleVotes.jsx]: I used section

File: src/components/ArticleVotes.jsx (hunk 1 above)
Comment (omobayed) [reply src/components/ArticleVotes.jsx]: done

**Pull request merged**